src/probnum/diffeq/_probsolve_ivp.py

```diff
@@ -174,11 +174,11 @@ def probsolve_ivp(
     Examples
     --------
     >>> from probnum.diffeq import probsolve_ivp
-    >>> from probnum import randvars
     >>> import numpy as np
 
     Solve a simple logistic ODE with fixed steps.
 
+    >>>
```

Comment:
Was this empty line intentional? It looks ok in the docs, but I was just wondering.

Reply:
yup